### PR TITLE
Replace Some Methods in Sklearn Example

### PR DESCRIPTION
**Pull request by MasahitoKumada**
## Motivation

related to  https://github.com/optuna/optuna-examples/issues/98

## Description of the changes

Replace
- LogUniformDistribution with FloatDistribution,
- IntUniformDistribution with IntDistribution

in the [sklearn example](https://github.com/optuna/optuna-examples/blob/4e65f19f4ad64f1234dba62d0fba1813194ad8fb/sklearn/sklearn_optuna_search_cv_simple.py#L18-L21).

### CODE REVIEW

File: sklearn/sklearn_optuna_search_cv_simple.py [hunk 1]
```diff
@@ -16,8 +16,8 @@
     clf = SVC(gamma="auto")
 
     param_distributions = {
-        "C": optuna.distributions.LogUniformDistribution(1e-10, 1e10),
-        "degree": optuna.distributions.IntUniformDistribution(1, 5),
+        "C": optuna.distributions.FloatDistribution(1e-10, 1e10),
```
Comment (toshihikoyanase) [inline sklearn/sklearn_optuna_search_cv_simple.py:19]: Could you use the `log` option of `FloatDistribution`, please?
https://optuna.readthedocs.io/en/latest/reference/generated/optuna.distributions.FloatDistribution.html#optuna.distributions.FloatDistribution.log

Comment (MasahitoKumada) [reply sklearn/sklearn_optuna_search_cv_simple.py]: Thank you for your comment. I have taken care of it.

Comment (toshihikoyanase) [reply sklearn/sklearn_optuna_search_cv_simple.py]: Great! Thanks.

**Review by toshihikoyanase — APPROVED** [review]
LGTM! Thank you for your contribution.

**Pull request merged**